packages/core/src/IndexedT/index.ts

```diff
@@ -207,57 +26,70 @@ export class Ix<I, O, A> {
   constructor(readonly value: A) {}
 }
 
+export interface IndexedT<F extends P.HKT, _I, _O> {
```

Comment:
@mikearnaldi I removed the initial Indexed representation and exposed directly an IndexedT, let me know if that's an issue and I'll rework it :).

(also, this type is pretty cool and simple! I got lost in the types with the old version but the new system makes what happens very explicit)